src/services/mtn-momo/client.ts

```
@@ -35,16 +37,60 @@ export class MTNMoMoClient implements FintechProvider {
+    try {
+      let attempt = 0;
+      while (attempt <= retries) {
+        try {
+          const result = await requestFn();
+          this.breaker.recordSuccess();
+          return result;
+        } catch (error: any) {
+          attempt++;
+          const isNetworkError = !error.response;
+          const isServerError = error.response?.status >= 500;
+          
```

ESLint

[error] Delete `··········`

(prettier/prettier)